consensus/scp/tests/test_mesh_networks.rs

```diff
@@ -63,49 +31,36 @@ fn mesh_1(logger: Logger) {
 
 #[test_with_logger]
 #[serial]
-fn mesh_2_k1(logger: Logger) {
+fn mesh_2k1(logger: Logger) {
     mesh_test_helper(2, 1, logger);
 }
 
 #[test_with_logger]
 #[serial]
-fn mesh_3_k1(logger: Logger) {
+fn mesh_3k1(logger: Logger) {
     mesh_test_helper(3, 1, logger);
 }
 
 #[test_with_logger]
 #[serial]
-fn mesh_3_k2(logger: Logger) {
+fn mesh_3k2(logger: Logger) {
     mesh_test_helper(3, 2, logger);
 }
 
 #[test_with_logger]
 #[serial]
-fn mesh_4_k3(logger: Logger) {
+fn mesh_4k3(logger: Logger) {
     mesh_test_helper(4, 3, logger);
 }
 
 #[test_with_logger]
 #[serial]
-fn mesh_5_k3(logger: Logger) {
+fn mesh_5k3(logger: Logger) {
     mesh_test_helper(5, 3, logger);
 }
 
 #[test_with_logger]
 #[serial]
-fn mesh_5_k4(logger: Logger) {
+fn mesh_5k4(logger: Logger) {
     mesh_test_helper(5, 4, logger);
 }
-
-// This is a very slow test :(
```

Comment:
This now takes my laptop about two and a half minutes in debug mode on master, so it may be worth keeping. Up to you.